src/check.sh

```diff
@@ -0,0 +1,5 @@
+IFS=
+while read -r a; do
+	# Doesn't really work in general (multiline comments are broken, ...), but works well enough
+	printf "%s  %s\n" "$(echo "$a" | sed -re 's/\s*([^a-zA-Z_])\s*/\1/g;s/.*\*\/|\/[/*].*|^\s*|\s*$//g' | md5sum | head -c3)" "$a"
```

Comment:
Tady by to asi chtělo komentář k tomu co to dělá, není to z toho moc jasný. Taky bychom to mohli zjednodušit tím, o čem jsme se bavili - tj prostě odstranit mezery (a taby, dokud je tam ještě máme 😛) a všechno za `//`